src/error.rs

```diff
@@ -87,9 +87,16 @@ pub(crate) enum Error<'src> {
   InitExists {
     justfile: PathBuf,
   },
+  IncludeRecursive {
+    cur_path: PathBuf,
+    recursively_included_path: PathBuf,
+  },
```

Comment:
I think we can improve this error. If it only prints the current path, and the path that we've already seen, it might not be clear why we've already seen this path. Check out `CompileError::{CircularRecipeDependency, CircularVariableDependency}` for example formatting.

The ideal form would be something like:

```suggestion
  CircularInclude {
    prefix: Vec<PathBuf>,
    cycle: Vec<PathBuf>,
  },
```

`prefix` is the includes that got us to this point, and `cycle` is the circular inclusion that triggered the error.

The prefix helps the user find where the cycle started, and the cycle helps the user figure out what the actual cycle was.